docs/docs/technology/awards.md

```diff
@@ -1,33 +1,23 @@
 # Awards and Recognition
 
-In this page, we list the awards and recognition that the technology behind sign.mt has received in the last few years, alongside public independent reviews.
+In this page, we list the awards and recognition that the technology behind sign.mt and Dr. Amit Moryossef has received in the last few years, alongside public independent reviews.
 
-## 2024
-
-- We received an `EMNLP Outstanding Demo Paper Award` for our demonstration of "sign.mt: Real-Time Multilingual Sign Language Translation Application"[^emnlp2024-award].
-
-[^emnlp2024-award]: sign.mt. 2024. [Twitter Post](https://x.com/signmt_/status/1857181686045540787).
-
-- Dr. Amit Moryossef (among others) received the `SwissNLP Award` for his "outstanding contribution to Sign Language Translation in Switzerland"[^swissnlp2024-award].
+Timothy Comment: Not ideal for reader/user, needs to read the award in the top, then scroll down (or click the number for source) then click the source down below again. One click to much. 
```

Comment:
this is not necessary here. if you want to leave a comment, use github, the conversation tab, or just a comment on the code.